guides/README.md

```diff
@@ -219,19 +232,22 @@ See the [reference template](https://raw.githubusercontent.com/redhat-documentat
 * [`snip`](https://redhat-documentation.github.io/modular-docs/#using_text_snippets_or_text_fragments_writing-mod-docs): Files starting with `snip_` contain snippets that are reused throughout multiple guides, e.g. admonitions.
 Snippets do not require an ID.
 
-### Images
-
-Each guide must have an `images/` subdirectory with `images/common` symlink into the `common/images/` directory.
-Images local to the guide shall be kept in the `images/` directory.
-Images which are supposed to be reused across guides shall be kept in the `images/common/` directory.
-Subdirectories can be created and are actually recommended.
+### Creating new guides
 
-To insert an image, use `image::common/global_image.png` or `image::local_image.png`.
+Each guide must be in a separate directory in guides/ prefixed with "doc-". The following requirements must be met:
 
-You should create upstream diagrams using [diagrams.net](https://www.diagrams.net/).
-Place the editable diagram in `drawio` format in `guides/image-sources/`.
-For inclusion in the content, export diagrams to SVG and place them as described above.
+* Top-level file `master.adoc`.
+* Directory `topics` with actual content.
+* Symlink `common` to `../common` for shared content.
+* Directory `images` with images.
+* Symlink `images/common` to `../common/images`.
+* File `Makefile` which includes `../common/Makefile`.
+* File `docinfo.xml` for downstream (Red Hat) guides. Presence of this file will cause build check via `ccutil`, a tool used by Red Hat documentation team.
+* Put new content into the `guides/common/modules` directory and stick to one (level one) heading per file.
+* Update `web/content/js/nav.js` to add a new item on the top-level menu.
 
+New guides can be typically created as copies of existing guides, just make sure to clean images, topics and `docinfo.xml`.
 
+### Reviewing
 
-For more information, see the [Modular Documentation Reference Guide](https://redhat-documentation.github.io/modular-docs/).
+If the review would result in many comments in a GitHub pull request, consider checking out the topic branch and putting necessary changes on top of author's work instead of making the comments.
```

Comment:
I am generally OK with this, but don't practice it myself. Let's talk about this next meeting (not blocking this PR!)

Reply:
https://github.com/theforeman/foreman-documentation/pull/2742